### PR TITLE
Work on Options

### LINKED ISSUE #387: KDE system tray menu issues
This is on Kubuntu 18.04 (Psi v1.3, Qt 5.9.5):
psi_menu_bug
The menu is displayed out of the srcreen. If you hover over menu positions you can also notice that sometimes (second hover on the same entry) it will display correctly, but sometimes it does not display the submenu at all.

### PR DESCRIPTION
**Pull request by Vitozz**
menu_account_setup action removed.
Removed submenus from tray menu.
Pinned tab caption size increased from 6 letters to 12.
Can fix #387

### CODE REVIEW

File: src/options/opt_accounts.cpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{
+}
+
+OptionsTabAccounts::~OptionsTabAccounts()
+{
+}
+
+QWidget *OptionsTabAccounts::widget()
+{
+    if (w_) {
```
Comment (Ri0n) [inline src/options/opt_accounts.cpp]: This doesn't look right

File: src/psicon.cpp [hunk 2]
```diff
@@ -954,28 +953,6 @@ void PsiCon::changeProfile()
     doQuit(QuitProfile);
 }
 
-void PsiCon::doManageAccounts()
-{
-    if (!PsiOptions::instance()->getOption("options.ui.account.single").toBool()) {
```
Comment (Ri0n) [inline src/psicon.cpp]: probably for single acount setup we could render acount modify diaog right into options. 
and if there is no any account yet, then just a label with prompt and button to create one.

File: src/options/opt_accounts.cpp (hunk 1 above)
Comment (Vitozz) [reply src/options/opt_accounts.cpp]: What do you mean? In other option tabs the same functions with the same if statements.

File: src/psicon.cpp (hunk 2 above)
Comment (Vitozz) [reply src/psicon.cpp]: Maybe. But account modify dialog has its own dialog buttons and we probably need to hide this buttons somehow. And also send to modify dialog signals from dialog buttons of options dialog.

Comment (Ri0n) [reply src/psicon.cpp]: to continue a chat discussion..
Well I don't mind to remove this option at all. But I can imagine a business  case when Psi set up the way making it difficult to modify options (e.g. for a company's employees), and this one seems to be useful then.

@jkarneges  do you remember what this option for?

File: src/options/opt_accounts.cpp (hunk 1 above)
Comment (Ri0n) [reply src/options/opt_accounts.cpp]: oh. I see. I misintepreted the code. I thought it has to be `if(!w_)`

File: src/psicon.cpp (hunk 2 above)
Comment (jkarneges) [reply src/psicon.cpp]: @Ri0n I think that's exactly the reason. Most business versions/forks of Psi have been single account.

**Comment by Vitozz** [comment]
That's how it looks like
pic-20190916-201108
pic-20190916-201150
pic-20190916-201232

**Comment by tehnick** [comment]
Looks great!

**Comment by Ri0n** [comment]
push

**Comment by Vitozz** [comment]
Already pushed :) one commit ago

File: src/psicon.cpp (hunk 2 above)
Comment (Vitozz) [reply src/psicon.cpp]: Single account option support implemented with disabling AccountManageDlg elements. Screenshots added below.

**Pull request merged**